### PR TITLE
Create simple bot with botkit

### PR DESCRIPTION
**Pull request by solairerove**
final and release version of kit intergration

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,12 @@
 {
-    "name": "slack-bot-demo",
+    "name": "slackbot-vitalka",
     "version": "0.0.1",
-    "description": "Demo bot for slack",
+    "description": "Meme bot for slack",
     "main": "app.js",
     "author": "Nikita Krivitski",
```
Comment (vlsidlyarevich) [inline package.json]: Not only Krivitski, pidor

File: package.json [hunk 2]
```diff
@@ -1,14 +1,12 @@
 {
-    "name": "slack-bot-demo",
+    "name": "slackbot-vitalka",
     "version": "0.0.1",
-    "description": "Demo bot for slack",
+    "description": "Meme bot for slack",
```
Comment (vlsidlyarevich) [inline package.json]: And time

**Review by vlsidlyarevich — COMMENTED** [review]
Fix commented issues, fucker

**Review by solairerove — COMMENTED** [review]
look here blyat

**Review by vlsidlyarevich — APPROVED** [review]
Nice work

**Pull request merged**